### PR TITLE
Fix toString methods to display correct message upon adding a policy and deleting a policy

### PR DESCRIPTION
**Pull request by youdonnnn**

### CODE REVIEW

File: src/main/java/seedu/address/model/person/Person.java [hunk 1]
```diff
@@ -134,8 +134,7 @@ public int hashCode() {
      */
     @Override
     public String toString() {
-        return new ToStringBuilder(this)
-                .add("name", name)
+        return new ToStringBuilder(name.toString())
```
Comment (jimseah-0116) [inline src/main/java/seedu/address/model/person/Person.java:137]: Good catch!

**Review by jimseah-0116 — APPROVED** [review]
LGTM!

File: src/main/java/seedu/address/commons/util/ToStringBuilder.java [hunk 2]
```diff
@@ -4,10 +4,8 @@
  * Builds a string representation of an object that is suitable as the return value of {@link Object#toString()}.
  */
 public class ToStringBuilder {
-    private static final String OBJECT_PREFIX = "{";
-    private static final String OBJECT_SUFFIX = "}";
     private static final String FIELD_SEPARATOR = ", ";
-    private static final String FIELD_NAME_VALUE_SEPARATOR = "=";
+    private static final String FIELD_NAME_VALUE_SEPARATOR = ": ";
```
Comment (irwynliong) [inline src/main/java/seedu/address/commons/util/ToStringBuilder.java:8]: good standardization!

**Review by irwynliong — APPROVED** [review]
Thankers! 😃

**Pull request merged**